src/patternfly/components/FormControls/styles.scss

```diff
@@ -11,17 +35,31 @@
   --pf-c-form-control--PaddingBottom: calc(#{var(--pf-global--spacer--sm)} - 3px);
   --pf-c-form-control--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-form-control--Color: var(--pf-global--Color--dark-100);
-  --pf-c-form-control--disabled--Color: var(--pf-global--Color--disabled);
   --pf-c-form-control--BorderSize: var(--pf-global--BorderWidth--sm);
-  --pf-c-form-control--focus--BorderSize: var(--pf-global--BorderWidth--md);
-  --pf-c-form-control--disabled--BorderSize: var(--pf-global--BorderWidth--sm);
   --pf-c-form-control--BorderColor: var(--pf-global--BorderColor--dark);
   --pf-c-form-control--BorderRadius: var(--pf-global--BorderRadius--sm);
-  --pf-c-form-control--disabled--BorderColor: var(--pf-global--BorderColor--disabled);
   --pf-c-form-control--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+
+  // input placeholder style
+  --pf-c-form-control--placeholder--Color: var(--pf-global--Color--200);
+
+  // input disabled style
+  --pf-c-form-control--disabled--Color: var(--pf-global--Color--disabled);
+  --pf-c-form-control--disabled--BorderSize: var(--pf-global--BorderWidth--sm);
+  --pf-c-form-control--disabled--BorderColor: var(--pf-global--BorderColor--disabled);
   --pf-c-form-control--disabled--BackgroundColor: var(--pf-global--BackgroundColor--disabled);
-  --pf-c-form-control--placeholder--Color: var(--pf-global--Color--dark-200);
-  --pf-c-form-control--m-login--focus--BottomBorderWidth: var(--pf-global--BorderWidth--md);
+
+  // input alt style
+  --pf-c-form-control--m-alt--focus--BottomBorderWidth: var(--pf-global--BorderWidth--md);
+
+  // Input m-error
+  --pf-c-form-control--error--PaddingTop: calc(#{var(--pf-global--spacer--sm)} - 4px);
+  --pf-c-form-control--error--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-form-control--error--PaddingBottom: calc(#{var(--pf-global--spacer--sm)} - 4px);
```

Comment:
We need a new value for padding-left too, so that the text is perfectly aligned in all the fields.

Also, we handle borders for buttons differently. Did we decide that method wouldn't work for inputs?

Reply:
good call, I'll update. Ya we couldn't use the same approach with inputs b/c you can't use pseudo elements.